src/materials/Material.js

```diff
@@ -255,6 +255,7 @@ Material.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		}
 
 		if ( this.size !== undefined ) data.size = this.size;
+		if ( this.shadowSide !== undefined ) data.shadowSide = this.shadowSide;
```

Comment:
Wait. I think it should be:
```js
if ( this.shadowSide !== null ) data.shadowSide = this.shadowSide;
```

Reply:
All  materials have a `shadowSide` property so it is never `undefinded`.

Reply:
Thanks a lot for checking so quickly. And you are very correct! I've updated the branch on my fork, does the PR automatically update from the remote branch or is there something I should do to update this PR?

Reply:
Yes, updating your branch is sufficient.

Reply:
Great, thank you